spark/src/main/scala-spark-3.5/shims/TypeWideningShims.scala

```diff
@@ -42,12 +42,7 @@ object TypeWideningShims {
       case _ => false
     }
 
-  /**
-   * Returns whether the given type change can be applied during schema evolution. Only a
-   * subset of supported type changes are considered for schema evolution.
-   */
-  def isTypeChangeSupportedForSchemaEvolution(fromType: AtomicType, toType: AtomicType): Boolean = {
```

Comment:
There are no types changes anymore that can only be applied via `ALTER TABLE` but not during schema evolution, this method is removed.